zeebe/gateway/src/main/java/io/camunda/zeebe/gateway/impl/broker/request/group/BrokerGroupMemberRequest.java

```diff
@@ -18,22 +18,24 @@ public class BrokerGroupMemberRequest extends BrokerExecuteCommand<GroupRecord>
 
   private final GroupRecord requestDto = new GroupRecord();
 
-  public BrokerGroupMemberRequest(final long groupKey, final GroupIntent intent) {
+  public BrokerGroupMemberRequest(final String groupId, final GroupIntent intent) {
     super(ValueType.GROUP, intent);
-    request.setKey(groupKey);
-    requestDto.setGroupKey(groupKey);
+    final var key = Long.parseLong(groupId);
+    request.setKey(key);
```

Comment:
🔧 Could you ensure this gets send to the deployment partition. I guess it might be part of a different issue just like the other ones.

Reply:
This will be refactored in another issue 👍